packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -67,6 +67,16 @@ export interface AttesationServiceRevealRequest {
   salt?: string
   smsRetrieverAppSig?: string
   language?: string
+  securityCodePrefix?: string
+}
+
+export interface AttestationServiceSecurityCodeRequest {
```

Comment:
it would probably make sense to use the same `GetAttestationRequestType` in utils that is used by attestation service. could also update the `AttesationServiceRevealRequest` to the `AttestationRequestType`